brozzler/__init__.py

```diff
@@ -68,29 +68,34 @@ def _logger_trace(self, msg, *args, **kwargs):
 logging._nameToLevel['TRACE'] = TRACE
 
 _behaviors = None
-def behaviors():
+def behaviors(behaviors_dir=None):
+    """Return list of JS behaviors loaded from YAML file.
+
+    :param behaviors_dir: Directory containing `behaviors.yaml` and
+    `js-templates/`. Defaults to brozzler dir.
+    """
     import os, yaml, string
     global _behaviors
     if _behaviors is None:
-        behaviors_yaml = os.path.join(
-                os.path.dirname(__file__), 'behaviors.yaml')
+        cwd = behaviors_dir or os.path.dirname(__file__)
```

Comment:
`cwd` is a misleading name for this variable... could use `d`

Reply:
OK, fixed.